inst/retentionIndex/metabolites.txt

```diff
@@ -0,0 +1,77 @@
+name	rtime	rindex_manual
+VITAMIN D2	NA	NA
+SQUALENE	15.66	1709.876543
```

Comment:
Hm, I find it a little unintuitive to call a floating point number *index* - I thought index would be an `integer`? Wouldn't it be *better* to provide integer indices instead?

Reply:
It is not like an index as you might think of, e.g. from a database. It's more like a conversion to a different "unit". Most people use it as integers, but I would like to keep it as floating point number and let the user decide.

Reply:
Fine for me - maybe you could describe this also in the documentation?